src/Components/SmartComponents/SystemsExposedTable/SystemsExposedTable.test.js

```diff
@@ -117,15 +117,4 @@ describe('SystemsExposedPage', () => {
         const dispatchedActions = store.getActions();
         expect(dispatchedActions.filter(item => item.type === 'FETCH_AFFECTED_SYSTEMS_BY_CVE')).toHaveLength(1);
     });
-
-    it('Should update inventory', () => {
-        const testObj = { page: 2, per_page: 2}
-        wrapper.update(); 
-        act( () => wrapper.find('InventoryTable').props().onRefresh(testObj)); 
```

Comment:
I think this is not connected with the snapshots but there is no `wrapper.find('InventoryTable')` 
try instead `wrapper.find('BaseInvTable)`.